konig-spreadsheet/src/main/java/io/konig/spreadsheet/WorkbookLoader.java

```diff
@@ -1029,11 +1066,282 @@ private void loadSheet(SheetInfo info) throws SpreadsheetException {
 			case SHEET_SECURITY_TAGS:
 				loadSecurityTags(sheet);
 				break;
+			case SHEET_DATA_DICTIONARY_TEMPLATE:
+				loadDataDictionary(sheet);
+				break;
 				
 			}
 
 		}
 
+		private void loadDataDictionary(Sheet sheet) throws SpreadsheetException {
+			readDataDictionaryTemplateHeader(sheet);
+
+			int rowSize = sheet.getLastRowNum() + 1;
+			for (int i = sheet.getFirstRowNum() + 1; i < rowSize; i++) {
+				Row row = sheet.getRow(i);
+				loadDataDictionaryTemplateRow(row);
+			}
+			
+		}
+
+		private void loadDataDictionaryTemplateRow(Row row) throws SpreadsheetException {
+			String sourceSystemName = stringValue(row, sourceSystemCol);
+			String shapeIdLocalName = stringValue(row, sourceObjectNameCol);
+			if(shapeIdLocalName==null )
+				return;
+			URI shapeId = getShapeURL(sourceSystemName,shapeIdLocalName);
+			if (shapeId == null)
+				return;
+			String propertyIdValue = stringValue(row, fieldCol);
+			if (propertyIdValue == null) {
+				logger.warn("Shape Id is defined but Property Id is not defined: {}", shapeId.getLocalName());
+				return;
+			}
+			URI propertyId = expandPropertyId(getPropertyBaseURL()+propertyIdValue);
+
+			logger.debug("loadPropertyConstraintRow({},{})", RdfUtil.localName(shapeId), RdfUtil.localName(propertyId));
+
+			String dataDictionarydataType=stringValue(row,dataTypeCol);
+
+			Literal maxLength = numericLiteral(row, maxLengthCol);
+			Literal decimalPrecision = intLiteral(row, decimalPrecisionCol);
+			if(decimalPrecision!=null && (decimalPrecision.intValue()<1 || decimalPrecision.intValue()>65)){
+				throw new KonigException("Decimal Precison should be between 1 to 65");
+			}
+			Literal decimalScale = intLiteral(row, decimalScaleCol);
+			if(decimalScale!=null && 
+					(decimalScale.intValue()<0 || decimalScale.intValue()>30 || 
+							decimalScale.intValue()>decimalPrecision.intValue())){
+				throw new KonigException("Decimal Scale should be less than Decimal Precision and between 0-30");
+			}
+			Literal businessName = stringLiteral(row, businessNameCol);
+			Literal businessDefinition = stringLiteral(row, businessDefinitionCol);
+			Literal dataSteward = stringLiteral(row, dataStewardCol);
+			String securityClassification=stringValue(row,securityClassifCol);
+			String constraints = stringValue(row,constraintsCol);			
+			URI rdfSecurityClassification=getRdfSecurityClassification(securityClassification);
```

Comment:
The rdfSecurityClassification is not needed since the users must supply a fully-qualified URI or CURIE (as mentioned above).